jest.config.ts

```diff
@@ -69,6 +69,9 @@ const config: Config.InitialOptions = {
 	],
 	testEnvironment: 'jsdom',
 	...(isRunningInEsmMode && esmConfig),
+	moduleNameMapper: {
+		'^@scripts/(.*)$': '../../../scripts/$1',
```

Comment:
this needs to be rejigged a bit as it's not working for the server.ts tests here

Reply:
Removing this from the PR and alias logic will be completed in another task